### DISCUSSION TITLE
Checkout API

### DISCUSSION THREAD

**Discussion by olivermrbl** [medusajs/medusa#2851] [Feature Requests]
### What
In our efforts to support a broader range of payment providers, we will introduce a Checkout API defining the interface for payment providers that combines authorization and capture in a single step e.g. PayPal, GooglePay, ApplePay.

**Comment by michacassola** [comment]
Will this also make integrating Paddle supported?

**Comment by noureddine-as** [comment]
Are there any plans to support this feature?
